lib/metadataTypes/MetadataType.js

```diff
@@ -287,29 +300,42 @@ class MetadataType {
                             'Hotfix for non-cachable resource found in deploy folder. Trying update:'
                         );
                         Util.logger.warn(JSON.stringify(metadata[metadataKey]));
-                        metadataToUpdate.push({
-                            before: {},
-                            after: metadata[metadataKey],
-                        });
+                        if (error) {
+                            metadataToUpdate.push(null);
```

Comment:
why would we push null?

Reply:
described in the PR text:
to update the "the total counter for created / updated "
I wanna see 0 of 1 instead of 0 of 0 there.